docs/Upgrade.rst

```diff
@@ -51,6 +51,28 @@ In this section, we will present all the breaking changes you should
 know before starting the upgrade to 6.0 version.
 
 
+**Architecture changes**
+
+The components architecture has changed in 6.0 version:
+
+- ECMS, JCR, Forum and Calendar components are considered now as addons and are no more pre-installed in community edition.
+- Forum addon has been deprecated and its support will be removed in future versions.
+- JCR is not considered anymore as a basic component of the platform, and thus must be installed to be used.
+- Chromattic library has been deleted from pre-packaged bundle.
+- intranet site has been deprecated and moved to `exo-legacy-intranet <https://github.com/exoplatform/legacy-intranet>` addon 
+- `integration <https://github.com/exoplatform/integration>` is not used since 6.0 anymore
+- `platform <https://github.com/exoplatform/platform>` is not used since 6.0 anymore
+- `doc-style <https://github.com/exoplatform/doc-style>` is not used since 6.0 anymore
+- Layout management features has been moved to `exo-layout-management <https://github.com/exoplatform/layout-management>` addon
+- Data upgrade plugins are moved to `exo-data-upgrade <https://github.com/exoplatform/data-upgrade>` addon
+- To Upgrade data, the addon `exo-data-upgrade <https://github.com/exoplatform/data-upgrade>` addon must be installed manually
```

Comment:
This should be moved to the upgrade process chapter.

Reply:
yep, it will be moved to the next chapter (Upgrade process)  once the upgrade process is reworked (kept here temporarily)

Reply:
done